### DISCUSSION TITLE
Jellyfin autoupdate to 10.9, SSO plugin no longer working

### DISCUSSION THREAD

**Discussion by mobiledude** [9p4/jellyfin-plugin-sso#188] [Q&A] [answered]
Will there be an update for 10.9 considering Jellyfin just released their 10.9 version and it does not seem to work anymore

**Comment by 9p4** [comment, marked as answer]
I'm on Jellyfin 10.9 in a container, and the SSO plugin 3.5.2.3. The plugin works fine. What's not working for you? What versions do you have installed?

**Reply by mobiledude** [reply]
Thank you for confirming that it should work! after I deleted my cookies, it worked again!
